package.json

```diff
@@ -21,8 +21,7 @@
     "depcheck": "node ./scripts/depcheck.js"
   },
   "dependencies": {
-    "@accordproject/cicero-core": "^0.13.4",
-    "@accordproject/markdown-editor": "^0.7.2",
+    "@accordproject/markdown-editor": "^0.7.1",
```

Comment:
why are we downgrading markdown-editor here?

Reply:
Unsure... This will be updated when a new `markdown-editor` is released after this PR: https://github.com/accordproject/markdown-editor/pull/188

Reply:
👍 Let's put it back to 0.7.2 for now, bc I think this PR looks good and can be merged otherwise.